python/tests/unit/updater/test_standard.py

```diff
@@ -0,0 +1,65 @@
+"""Test standard."""
```

Comment:
Typo: multiple within this file; I think it is `reduced` as opposed to `reduce`.

Reply:
https://github.com/benjamincerigo/GildedRose-Refactoring-Kata/pull/1/commits/4da6aa405bdf5cc811634779e2db808547fb3d0c